appdata.patch

```diff
@@ -0,0 +1,62 @@
+diff --git a/data/mx.pwmc.Svgvi.appdata.xml.in b/data/mx.pwmc.Svgvi.appdata.xml.in
+index 62ac7f8..7c7522d 100644
+--- a/data/mx.pwmc.Svgvi.appdata.xml.in
++++ b/data/mx.pwmc.Svgvi.appdata.xml.in
+@@ -11,10 +11,13 @@
+   <project_license>GPL-3.0-or-later</project_license>
+   <name>Svgvi</name>
+   <summary>SVG viewer and editor using XML declarations</summary>
++  <developer id="tld.domain">
```

Comment:
Should be an actual reverse domain

Reply:
Is the ID an identifier of the application?
Do I set mx.pwmc.Svgvi?

Reply:
No, a domain owned by the developer. You can use `mx.pwmc`

Reply:
I see.
```
mx.pwmc
```

Reply:
Thanks for the advice.
I will wait until the build test is done.

Reply:
The test appears to have passed.
Thanks a lot.
:blush: